### PR TITLE
refactor: added separate app for advertising (#18)

### PR DESCRIPTION
**Pull request by yspanchal**
[refactor] Separate out application of `advertising` Issue: https://github.com/fafadiatech/newscout_web/issues/18

### CODE REVIEW

**Comment by sidharthshah** [comment]
@yspanchal is this good to go?

**Comment by yspanchal** [comment]
> @yspanchal is this good to go?

yes.

File: newscout_web/advertising/urls.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from .views import (GetAds, CampaignView,
+                    CampaignDeleteView, AdGroupView, AdGroupDeleteView,
+                    AdvertisementView, AdvertisementDeleteView,
+                    CampaignCategoriesListView, GroupTypeListView,
+                    AdRedirectView)
+
+urlpatterns = [
+    url(r'^get-ads/$', GetAds.as_view(), name='get-ads'),
```
Comment (sidharthshah) [inline newscout_web/advertising/urls.py]: Rename this to `ads`

File: newscout_web/advertising/urls.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    url(r'campaign-categories/$', CampaignCategoriesListView.as_view(),
```
Comment (sidharthshah) [inline newscout_web/advertising/urls.py]: Rename this to `campaign/categories/`

File: newscout_web/newscout_web/urls.py [hunk 3]
```diff
@@ -26,6 +25,5 @@
     url(r'^admin/', admin.site.urls),
     url(r'^api/', include('api.urls')),
     url(r'^event/', include('event_tracking.urls')),
-    url(r'^getad-redirect/$', AdRedirectView.as_view(),
-        name='get-ads-redirect'),
+    url(r'^ad/', include('advertising.urls')),
```
Comment (sidharthshah) [inline newscout_web/newscout_web/urls.py]: Rename this to `ads`

**Pull request merged**